.travis.yml

```diff
@@ -11,7 +11,6 @@ install:
   - pip install Django==$DJANGO_VERSION coveralls
 
 env:
-  - DJANGO_VERSION=1.6.5
   - DJANGO_VERSION=1.7.0
   - DJANGO_VERSION=1.8.1
```

Comment:
Should we also bump the version of Django 1.8? Or is there a way to specify `DJANGO_VERSION` as a max? Something like:

```yml
install:
   - pip install -r requirements.txt
   - pip install -U Django<$DJANGO_VERSION coveralls
 
 env:
   - DJANGO_VERSION=1.8
   - DJANGO_VERSION=1.9
```